docs/dbaas/DBaaS_tepmplate.md

```diff
@@ -0,0 +1,165 @@
+# Create a database cluster from a template
+
+Database clusters can be created from templates using PMM. Templates allow you to customize the creation of database clusters. You can adapt templates to tweak K8s-specific settings such as **liveness probes**, changing **config maps**, or tuning **database engines**. 
+
+## Create Custom Resource Definition (CRD) template
+
+To create a template, do the following:
+
+
+1. Change the `updateStrategy` and `upgradeOptions` fields as per the [PXC operator documentation](https://docs.percona.com/percona-operator-for-mysql/pxc/update.html#manual-upgrade_1) and [PXC CRD](https://github.com/percona/percona-xtradb-cluster-operator/blob/v1.11.0/deploy/crd.yaml#L8379-L8392).
```

Comment:
We should mention that this is just an example . It shows how one would go about changing the `upgradeOptions` field but it would be different if we wanted to customize some other field.

Reply:
I also think we should explicitly state that the Template CRD needs to have the a openAPIV3Schema that must be a subset of the parent engine CRD, in this case https://github.com/percona/percona-xtradb-cluster-operator/blob/v1.11.0/deploy/crd.yaml